packages/kbn-config/src/deprecation/deprecation_factory.test.ts

```diff
@@ -296,7 +302,8 @@ describe('DeprecationFactory', () => {
                   "Remove \\"myplugin.deprecated\\" from the Kibana config file, CLI flag, or environment variable (in Docker only)",
                 ],
               },
-              "message": "myplugin.deprecated is deprecated and is no longer used",
+              "message": "Setting \\"myplugin.deprecated\\" is deprecated and is no longer used",
```

Comment:
```suggestion
              "message": "You no longer need to configure this value. Remove \\"myplugin.deprecated\\" from the config.",
```

Reply:
The `message` is the description of the deprecation. I'd avoid adding a call to action here since we already detail what should be done in the manual steps displayed right below the deprecation `message` in the Upgrade assistant UI

Reply:
Agreed--it's redundant in this case. In general, I see the message as the What, and the steps as the How.